addons/website_event/views/event_templates.xml

```diff
@@ -124,73 +123,88 @@
-                            <span t-if="date[3]" class="badge badge-pill float-right"><t t-esc="date[3]"/></span>
-                        </a>
-                    </li>
-                </t>
-            </ul>
+        <div class="col-lg-3 css_noprint" id="left_column"/>
+    </xpath>
+</template>
+
+<template id="event_time" active="True" customize_show="True" inherit_id="website_event.event_left_column" name="Filter by Date">
+    <xpath expr="//div[@id='left_column']" position="inside">
+        <div class="card mb-3">
+            <div class="card-body">
+                <ul class="nav nav-pills flex-column">
+                    <t t-foreach="dates" t-as="date">
+                        <li class="nav-item" t-if="date[3] or (date[0] in ('old','all'))">
+                            <a t-attf-href="/event?{{ keep_query('country', 'type', date=date[0]) }}" t-attf-class="nav-link d-flex #{searches.get('date') == date[0] and ' active' or ''}"><t t-esc="date[1]"/>
+                                <span t-if="date[3]" class="badge badge-primary ml-auto align-self-center"><t t-esc="date[3]"/></span>
+                            </a>
+                        </li>
+                    </t>
+                </ul>
+            </div>
         </div>
     </xpath>
 </template>
 
 <template id="event_category" inherit_id="website_event.event_left_column" active="False" customize_show="True" name="Filter by Category">
     <xpath expr="//div[@id='left_column']" position="inside">
-        <ul class="nav nav-pills flex-column mt32">
-            <t t-foreach="types" t-as="type">
-                <li t-if="type['event_type_id']"
-                    class="nav-item">
-                    <a t-attf-href="/event?{{ keep_query('country', 'date', type=type['event_type_id'][0]) }}" t-attf-class="nav-link#{searches.get('type') == str(type['event_type_id'] and type['event_type_id'][0]) and ' active' or ''}"><t t-esc="type['event_type_id'][1]"/>
-                        <span class="badge badge-pill float-right"><t t-esc="type['event_type_id_count']"/></span>
-                    </a>
-                </li>
-            </t>
-        </ul>
+        <div class="card mb-3">
+            <div class="card-body">
+                <ul class="nav nav-pills flex-column">
+                    <t t-foreach="types" t-as="type">
+                        <li t-if="type['event_type_id']"
+                            class="nav-item">
+                            <a t-attf-href="/event?{{ keep_query('country', 'date', type=type['event_type_id'][0]) }}" t-attf-class="nav-link d-flex #{searches.get('type') == str(type['event_type_id'] and type['event_type_id'][0]) and ' active' or ''}"><t t-esc="type['event_type_id'][1]"/>
+                                <span class="badge badge-primary ml-auto align-self-center"><t t-esc="type['event_type_id_count']"/></span>
```

Comment:
Why not using `align-items-center` on the parent instead of `align-self-center` on all children?